### PR TITLE
Fully qualify the `with_server_config` helper

### PR DESCRIPTION
**Pull request by schisamo**
/cc @chefsalim @cwebberOps

### CODE REVIEW

File: .delivery/build-cookbook/recipes/deploy.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@
   require 'uri'
   include_recipe 'chef-sugar::default'
 
-  with_server_config do
+  DeliverySugar::ChefServer.new.with_server_config` do
```
Comment (cwebberOps) [inline .delivery/build-cookbook/recipes/deploy.rb]: you have a trailing ` that should go away

Comment (schisamo) [inline .delivery/build-cookbook/recipes/deploy.rb]: Hopefully Verify catches that. ;)

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by schisamo** [comment]
@delivery approve

**Comment by chef-delivery** [comment]
Change: 39e7eab6-62d6-4dc5-ac8a-7e977e1e6ae3 approved by: @schisamo

**Comment by chef-delivery** [comment]
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

**Comment by schisamo** [comment]
@delivery deliver

**Comment by chef-delivery** [comment]
Change: 39e7eab6-62d6-4dc5-ac8a-7e977e1e6ae3 delivered by: @schisamo

**Pull request merged**